packages/front-end/components/Experiment/TabbedPage/ExperimentHeader.tsx

```diff
@@ -206,33 +199,35 @@ export default function ExperimentHeader({
               )}
             </div>
 
-            <div className="ml-2">
-              {experiment.status === "running" ? (
-                <ExperimentActionButtons
-                  editResult={editResult}
-                  editTargeting={editTargeting}
-                />
-              ) : experiment.status === "stopped" && experiment.results ? (
-                <div className="experiment-status-widget border d-flex">
-                  <div
-                    className="d-flex"
-                    style={{ height: 30, lineHeight: "30px" }}
-                  >
-                    <ResultsIndicator results={experiment.results} />
+            {canRunExperiment ? (
```

Comment:
We weren't hiding the CTAs here if the user didn't have permission - so they could click, and only once they clicked would they get the permissions error. Throughout the app we simply don't show CTAs if the user doesn't have permission. This matches that pattern.